lib/cdo/script_constants.rb

```diff
@@ -274,11 +274,17 @@ module ScriptConstants
     DEEPDIVE_DEBUGGING = 'deepdive-debugging'.freeze,
     FREQUENCY_ANALYSIS = 'frequency_analysis'.freeze,
     GAMELAB = 'gamelab'.freeze,
+    HELLO_WORLD_FOOD = 'hello-world-food'.freeze,
+    HELLO_WORLD_ANIMALS = 'hello-world-animals'.freeze,
+    HELLO_WORLD_EMOJI = 'hello-world-emoji'.freeze,
+    HELLO_WORLD_RETRO = 'hello-world-retro'.freeze,
     K1HOC_2017 = 'k1hoc2017'.freeze,
     NETSIM = 'netsim'.freeze,
     ODOMETER = 'odometer'.freeze,
     OUTBREAK = 'outbreak'.freeze,
     PIXELATION = 'pixelation'.freeze,
+    POEM_ART = 'poem-art'.freeze,
+    POETRY_HOC3 = 'poetry-hoc3'.freeze,
```

Comment:
Do we still use `poetry-hoc3`?  It sounds a bit like a temporary name, but I'm not sure its history.

Reply:
I don't know the answers but I would be worried that we might lose translations by deleting it. We are planning to do a pass to deleted the old scripts from HOC so I think this can be cleaned up when we do